Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2090,7 +2090,9 @@ extension BrowserViewController {
     func getSignInOrFxASettingsVC(_ fxaOptions: FxALaunchParams? = nil, flowType: FxAPageType) -> UIViewController {
         // Show the settings page if we have already signed in. If we haven't then show the signin page
         guard profile.hasSyncableAccount() else {
-            return FxAWebView(pageType: flowType, profile: profile, dismissalStyle: .dismiss)
+            let signInVC = FirefoxAccountSignInViewController(profile: profile, parentType: .appMenu)
```

Comment:
nit: I believe this can be called from more than just the app menu, but this is ok as it is largest use-case